src/core/example_settings.py

```diff
@@ -95,6 +95,7 @@
 INSTALLED_APPS += plugin_installed_apps.load_homepage_element_apps()
 
 MIDDLEWARE_CLASSES = (
+    'utils.middleware.TimeMonitoring',
```

Comment:
I don't think this should be enabled by default... example settings could be used in production and this seems like a development thing.

Reply:
Done